ftests/rkt.go

```diff
@@ -87,3 +102,63 @@ func RunTestInContainer(t *testing.T, cfg RktConfig) {
 		t.Error(string(bytes))
 	}
 }
+
+// binLibs is a helper that prints libraries a given binary is dynamically
+// linked against, including recursive dependencies of those libraries.
+func binLibs(binPath string) ([]string, error) {
```

Comment:
This bit is a quite icky hack. It may be more desirable to rework the test code to use an environment similar to CL to run dynamically linked binaries rather than take this.

Reply:
Are you thinking about something like downloading a CL image and running tests in there via systemd-nspawn?

Reply:
Something like that would be nice. If we wanted to use docker, there's also a friendly user who [packaged the dev container up that way](https://github.com/BugRoger/coreos-developer-docker)